map2loop/mapdata.py

```diff
@@ -917,6 +906,20 @@ def parse_fault_map(self) -> tuple:
 
         # Create new geodataframe
         faults = geopandas.GeoDataFrame(self.raw_data[Datatype.FAULT]["geometry"])
+
+        # crop by minimum fault length
+        self.minimum_fault_length = 0.0
```

Comment:
replace with this self.minimum_fault_length = self.config.fault_config.get('minimum_fault_length',0.) unless printing the error message is important. I would suggest don't print in the middle of the code for config parameters but do a check of the config params when they are loaded.